source/reference/minio-mc-admin/mc-admin-prometheus-generate.rst

```diff
@@ -46,9 +52,11 @@ For more complete documentation on using MinIO with Prometheus, see :ref:`How to
       .. code-block:: shell
          :class: copyable
 
-         mc [GLOBALFLAGS] admin prometheus generate  \
-                                           ALIAS     \
-                                           [TYPE]
+         mc [GLOBALFLAGS] admin prometheus generate                                        \
+                                           ALIAS                                           \
+                                           [TYPE]                                          \
+                                           [--api_version v3]                              \
+                                           [TYPE --bucket <bucket name> --api_version v3]
```

Comment:
```suggestion
                                           [--bucket <bucket name>]
```

Save the explanation and v2/v3 differentiation for the explanation of the flags.